kubernetes/README.md

```diff
@@ -0,0 +1,86 @@
+# Deploying OpenWhisk on Kubernetes
+
+[![Build Status](https://travis-ci.org/openwhisk/openwhisk-devtools.svg?branch=master)](https://travis-ci.org/openwhisk/openwhisk-devtools)
+
+The collection of scripts and Kubernets pods can be used to
+run OpenWhisk in a Kubernets cluster.
+
+#### Kubernetes Requirments
+
+* Kubernetes needs to be version 1.5+
+* Kubernetes pods can receive public addresses
```

Comment:
It's not clear what this means. 
Assume that today it's the first day for the reader/user using this kubernetes  deployment. 

Does this mean I need to edit a config file? Or it means that the my computer where the kubernetes tool kubectl needs access to docker hub on the public internet?